### PR TITLE
[Test] Reconstruct benchmark & support fused ops

### PR DESCRIPTION
**Pull request by StrongSpoon**
1. move specific arg-gen functions into corresponding tests
2. add test_fused_perf.py (including gelu&mul, silu&mul, skip_layernorm, skip_rmsnorm)

### CODE REVIEW

File: benchmark/test_fused_perf.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+import pytest
+import flag_gems
+from .performance_utils import *
+
+
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_perf_gelu_and_mul(dtype):
+
+    def torch_op(x, y):
+        return torch.mul(torch.nn.functional.gelu(x), y)
```
Comment (tongxin) [inline benchmark/test_fused_perf.py:11]: Looks like `torch_op` is not used in the test right? This is just for reference purpose?

Comment (StrongSpoon) [reply benchmark/test_fused_perf.py:11]: yes. fused operators can't be called by torch api. so the compared operators are torch_op and gems_op.

Comment (tongxin) [reply benchmark/test_fused_perf.py:11]: I missed the baseline... It looks good.

**Review by tongxin — APPROVED** [review]
LG

**Review by iclementine — APPROVED** [review]

**Pull request merged**